ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -67,190 +64,165 @@ const ActiveRuntimeOp = ({operation, workspaceNamespace}) => {
-      } catch (e) {
-        // 404 is expected if the runtime doesn't exist, represent this as a null
-        // runtime rather than an error mode.
-        if (e.status !== 404) {
-          error = true;
+  return <div data-test-id='runtime-panel'>
+    <h3 style={styles.sectionHeader}>Cloud analysis environment</h3>
+    <div>
+      Your analysis environment consists of an application and compute resources.
+      Your cloud environment is unique to this workspace and not shared with other users.
+    </div>
+    {/* TODO(RW-5419): Cost estimates go here. */}
+    <div style={styles.controlSection}>
+      {/* Recommended runtime: pick from default templates or change the image. */}
+      <PopupTrigger side='bottom'
+                    closeOnClick
+                    content={
+                      <React.Fragment>
+                        <MenuItem style={styles.presetMenuItem}>General purpose analysis</MenuItem>
+                        <MenuItem style={styles.presetMenuItem}>Genomics analysis</MenuItem>
+                      </React.Fragment>
+                    }>
+        <Clickable data-test-id='runtime-presets-menu'
+                   disabled={true}>
+          Recommended environments <ClrIcon shape='caret down'/>
+        </Clickable>
+      </PopupTrigger>
+      <h3 style={styles.sectionHeader}>Application configuration</h3>
+      {/* TODO(RW-5413): Populate the image list with server driven options. */}
+      <Dropdown style={{width: '100%'}}
+                data-test-id='runtime-image-dropdown'
+                disabled={true}
+                options={[toolDockerImage]}
+                value={toolDockerImage}/>
+      {/* Runtime customization: change detailed machine configuration options. */}
+      <h3 style={styles.sectionHeader}>Cloud compute profile</h3>
+      <FlexRow style={{justifyContent: 'space-between'}}>
+        <MachineSelector updatedMachine={updatedMachine} onChange={setUpdatedMachine} masterMachineType={masterMachineType}/>
+        <DiskSizeSelection updatedDiskSize={updatedDiskSize} onChange={setUpdatedDiskSize} masterDiskSize={masterDiskSize}/>
+      </FlexRow>
+      <FlexColumn style={{marginTop: '1rem'}}>
+        <label htmlFor='runtime-compute'>Compute type</label>
+        <Dropdown id='runtime-compute'
+                  style={{width: '10rem'}}
+                  disabled={true}
+                  options={['Dataproc cluster', 'Standard VM']}
+                  value={isDataproc ? 'Dataproc cluster' : 'Standard VM'}/>
+      </FlexColumn>
+    </div>
+    <FlexRow style={{justifyContent: 'flex-end', marginTop: '.75rem'}}>
+      <Button
+        aria-label={currentRuntime ? 'Update' : 'Create'}
+        disabled={status !== RuntimeStatus.Running || !runtimeChanged}
+        onClick={() =>
+          setRequestedRuntime({dataprocConfig: {
+            masterMachineType: updatedMachineType || masterMachineType,
+            masterDiskSize: updatedDiskSize || masterDiskSize
```

Comment:
I would also pass `configurationType: RuntimeConfigurationType.UserOverride` ; later this will be derived from whether or not this matches one of the presets, but that can just be a TODO for now.